### ISSUE TITLE
Latest Postgres.app (2.7.4) release using 17beta1 Client Tools by default

### ISSUE THREAD

**Issue by PhilCoggins** [PostgresApp/PostgresApp#763]
Is there a particular reason Postgres.app chooses to use 17beta1 client tools by default? I believe I've encountered a bug (a snag, at the least) with the 17 client library. I thought it was an interesting choice to use beta client tools, I would expect it to use Stable by default.

**Comment by tbussmann** [comment]
thanks for your feedback! It's correct that the releases of 2.7.4 use the binaries of PostgreSQL 17 beta 1 at their 'latest' symlink. These are therefore marked as pre-releases on GitHub. The latest release for general use is still 2.7.3. However, on the homepage, this was not as clear as the "all+beta" bundle was listed before the "all" bundle. In a first step I've changed the order of the releases on the download page and will see how this can be improved further.

**Comment by PhilCoggins** [comment]
@tbussmann I appreciate the speedy response! Indeed, I downloaded the first release "All" on the downloads page and hadn't even seen that it included beta releases, so it makes sense now that 17 is chosen as the client library since 17 is the highest server version in that release (which is consistent with all Postgres.app releases). I uninstalled that app, installed the all supported versions release, and am back up-and-running. Feel free to close this issue.
